app/components/workshop-card.js

```diff
@@ -1,5 +1,11 @@
-import Ember from 'ember';
+import Ember from "ember";
+import moment from "moment";
 
 export default Ember.Component.extend({
-  tagName: ''
+  viewable: "Viewable Anytime",
```

Comment:
Can we use single quotes instead od double quotes for consistency with the rest of the codebase?